be/src/connector/hive_connector.cpp

```diff
@@ -424,6 +425,9 @@ Status HiveDataSource::_init_scanner(RuntimeState* state) {
     FSOptions fsOptions =
             FSOptions(hdfs_scan_node.__isset.cloud_configuration ? &hdfs_scan_node.cloud_configuration : nullptr);
 
+    // Use to print file path in be.out when coredump
+    tls_thread_status.set_lake_filename(native_file_path);
```

Comment:
is there any better name insteaf of lake_filename?

Reply:
Maybe more general name, like 'scan_file_path' or sth.

Reply:
fixed